.gitignore

```diff
@@ -63,18 +63,15 @@ docs/modules.rst
 docs/drf_rw_serializers.rst
 docs/drf_rw_serializers.*.rst
 
-# Private requirements
-requirements/private.in
-requirements/private.txt
-
 # tox environment temporary artifacts
 tests/__init__.py
 
 # Development task artifacts
 *.db
+Pipfile
```

Comment:
Is this necessary?

Reply:
The file is autogenerated during the `missing-migrations` step from `pre-commit`.

Reply:
Strange. Maybe pipenv is somehow still installed in your env?

Reply:
I will check again.

Reply:
I pushed a commit with a fix.